File: week-03/friday/07.py
# create a function that takes a list and returns a new list with all the elements doubled
list = [5,6,7,8,9]
new_list = []

def doubler(list):
    new_list = []
    for i in list:
        new_list.append(2*i)
    return new_list

print(doubler(list))

File: week-07/wednesday/07.js
'use strict';

var numbers = [2, 5, 11, 29];

// create a function that takes an array of numbers and returns a boolean
// it should return true if all the elements are prime, false otherwise

function isPrime(number) {
  for (var i = 2; i <= Math.floor(number/2); i++) {
    if (number % i === 0) {
      return false;
    }
  }
  return true;
}

function primeEveryChecker(array) {
  return array.every(function(e) {return isPrime(e);})
}

console.log(primeEveryChecker(numbers));
console.log(primeEveryChecker([3,5,7]));
